### PR TITLE
Added Visual Studio guards and fixed missing include

### PR DESCRIPTION
**Pull request by magnesj**

### CODE REVIEW

File: cmake/Modules/LibtoolArchives.cmake [hunk 1]
```diff
@@ -2,6 +2,10 @@
 # compiler/linker. first parameter is the name of the variable that will
 # receive this list, the rest is considered the list of libraries
 function (linker_cmdline what INTO outvar FROM)
+  if (NOT (UNIX OR MSYS OR MINGW))
+	return ()
+  endif (NOT (UNIX OR MSYS OR MINGW))
+
```
Comment (akva2) [inline cmake/Modules/LibtoolArchives.cmake]: :+1:

File: cmake/Modules/FindCXX11Features.cmake [hunk 2]
```diff
@@ -34,19 +34,21 @@ include(CheckIncludeFileCXX)
...
-  set(CXX_STD0X_FLAGS "-std=c++11")
-else()
-  # try to use compiler flag -std=c++0x for older compilers
-  CHECK_CXX_ACCEPTS_FLAG("-std=c++0x" CXX_FLAG_CXX0X)
-  if(CXX_FLAG_CXX0X)
-  add_options (CXX ALL_BUILDS "-std=c++0x")
-  set(CXX_STD0X_FLAGS "-std=c++0x")
-  endif(CXX_FLAG_CXX0X)
-endif(CXX_FLAG_CXX11)
+if(NOT MSVC)
```
Comment (akva2) [inline cmake/Modules/FindCXX11Features.cmake]: sadly this is necessary due to the cl.exe not returning an error. it simply prints a warning to terminal but exit code is 0.

Comment (andlaus) [inline cmake/Modules/FindCXX11Features.cmake]: I suppose this means that `CHECK_CXX_ACCEPTS_FLAG()` does not work at all for that compiler?

Comment (akva2) [inline cmake/Modules/FindCXX11Features.cmake]: aye :/

**Pull request merged**